docs/lab_08.html

```diff
@@ -98,12 +98,42 @@ <h2>Goals</h2>
 <ul>
 	<li>Learn how to commit changes to the repository</li>
 </ul>
-<h2>Commit the change</h2>
+
+<h2>Committing using an editor</h2>
 <p>Ok, enough about staging.  Let&#8217;s commit what we have staged to the repository.</p>
+<p>When you used <code>git commit</code> previously to commit the initial version of the <code>hello.js</code> file to the repository, you included the <code>-m</code> flag that gave a comment on the command line.  The commit command will allow you to interactively edit a comment for the commit.</p>
+<p>If you omit the <code>-m</code> flag from the command line, git will pop you into the editor of your choice.  The editor is chosen from the following list (in priority order):</p>
```

Comment:
Why not mention it when they commit on the previous step?


I think it could be written more clearly

```suggestion
<p>Previously, when committing the <code>hello.js</code> file, you entered the command <code>git commit -m &ldquo;Your commit message&rdquo;</code>. The <code>-m</code> flag will accept the commit message inline, within the &ldquo;double quotes&rdquo;. If you enter the command <code>git commit</code> without the <code>-m</code> flag, your default terminal editor will open </p>
<p>If you omit the <code>-m</code> flag from the <code>git commit</code> command, git will open your default terminal editor.</p> 
<p>The editor is chosen from the following list (in priority order):</p>
```